libpod/pod.go

```diff
@@ -70,6 +70,13 @@ type PodConfig struct {
 
 	// ID of the pod's lock
 	LockID uint32 `json:"lockID"`
+
+	// InfraImage is the image a pod infra container will use to manage
+	// namespaces.
+	InfraImage string `json:"infraImage,omitempty"`
```

Comment:
Can you take these out of PodConfig now that they're in InfraContainerConfig?

Reply:
Light dawns on marble head.  Fixed.